src/components/ha-metric.ts

```diff
@@ -0,0 +1,77 @@
+import { css, CSSResultGroup, html, LitElement, TemplateResult } from "lit";
+import { customElement, property } from "lit/decorators";
+import { classMap } from "lit/directives/class-map";
+import { roundWithOneDecimal } from "../util/calculate";
+import "./ha-bar";
+import "./ha-settings-row";
+
+@customElement("ha-metric")
+class HaMetric extends LitElement {
```

Comment:
Why is this file in `src/components` ? Doesn't seem like a component we're going to reuse.

Reply:
I would like start using this other places